wkconnect/webknossos/models.py

```diff
@@ -34,17 +36,27 @@ class DataSourceId(NamedTuple):
     name: str
 
 
-@dataclass(frozen=True)
+@dataclass(unsafe_hash=True)
 class DataLayer:
     name: str
     category: str
     boundingBox: BoundingBox
     resolutions: List[Vec3D]
     elementClass: str
+    dataFormat: str = field(default="wkw", init=False)
+    wkwResolutions: JSON = field(init=False)
+    largestSegmentId: Optional[int] = field(default=None, init=False)
+    mappings: Optional[List[Any]] = field(default=None, init=False)
 
     def __post_init__(self) -> None:
         assert self.category in ["color", "segmentation"]
         assert self.elementClass in ["uint8", "uint16", "uint32", "uint64"]
+        if self.category == "segmentation":
+            self.largestSegmentId = cast(int, np.iinfo(self.elementClass).max)
```

Comment:
not sure if that might cause trouble when creating volume tracings with segmentation fallback layer (since the space of remaining segmentIds becomes empty)? But that is, I suppose, open for future discussion